plugins/app-backend/package.json

```diff
@@ -57,7 +57,7 @@
     "test": "backstage-cli package test"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.25.0",
+    "@backstage/backend-defaults": "workspace:^",
```

Comment:
This isn't a dependency we can have. But I think it's fine to move back, I can't see any usage?

Reply:
You're right, I think I got it mixed up while comparing some of the other plugins,